terraform/groups/ecs-service/vault.tf

```diff
@@ -0,0 +1 @@
+ln -s vault-providers/userpass vault.tf
```

Comment:
Is this actually a symbolic link? Compare with https://github.com/companieshouse/certificates.orders.web.ch.gov.uk/blob/master/terraform/groups/ecs-service/vault.tf and https://github.com/companieshouse/identity-verification-api/blob/main/terraform/groups/ecs-service/vault.tf?